llvm/lib/Target/NVPTX/NVPTXCtorDtorLowering.cpp

```diff
@@ -32,6 +33,11 @@ static cl::opt<std::string>
               cl::desc("Override unique ID of ctor/dtor globals."),
               cl::init(""), cl::Hidden);
 
+static cl::opt<bool>
+    CreateKernels("nvptx-lower-global-ctor-dtor-kernel",
+                  cl::desc("Do not emit the init/fini kernels."),
```

Comment:
The description seems to contradict the option intent.

Reply:
Good catch, forgot to change it after adjusting its use.